### DISCUSSION TITLE
Manual releases Failing

### DISCUSSION THREAD

**Discussion by Jithu17** [projen/projen#1731] [Q&A]
Hi,

I am doing a manual release through CICD.
I am getting an error as ``Supplied version x.x.x was not found in input changelog. You may want to check it's content.``
Even though the input changelog in dist folder has the version that is required.

```
const inputChangelogContent = await readFile(inputChangelog, "utf-8");
  const changelogVersionSearchPattern = `[${version}]`;

  if (!inputChangelogContent.includes(changelogVersionSearchPattern)) {
    throw new Error(
      `Supplied version ${version} was not found in input changelog. You may want to check it's content.`
    );
  }
```

Should the changelogVersionSearchPattern = `${version}` rather than `[${version}]`
Apologies if I am missing something

**Comment by gradybarrett** [comment]
@Jithu17, it depends, but for the default config used in projen I think you're right. I'm not sure if this changed at some point. I could have sworn that when this feature was implemented the changelogs had brackets in the headers. Or maybe it just never worked... 🤷‍♂️

- [Default style changelog](https://github.com/conventional-changelog/conventional-changelog/blob/8076d4666c2a3ea728b95bf1e4e78d4c7189b1dc/packages/conventional-changelog-writer/templates/header.hbs) - note the missing brackets
- [Angular style](https://github.com/conventional-changelog/conventional-changelog/blob/8076d4666c2a3ea728b95bf1e4e78d4c7189b1dc/packages/conventional-changelog-angular/templates/header.hbs) - with brackets

Would be nice to be able to use brackets - every other template except the default one uses them - in order to limit the possibility of matching a string within the changelog that happened to match the current version. But for now, just dropping the brackets in the match should be fine.

**Reply by Jithu17** [reply]
@gradybarrett, thanks for the reply 
Now when the manual release the changelog has version with brackets, I am not sure whether the config was changed 
but now when I do a release the package-lock.json does not reverts back to 0.0.0 during unbump and hence the git diff step fails.

Is there any step that I can do to exclude that task or is there any way I can exclude tasks from release

Thanks

**Reply by gradybarrett** [reply]
Can you walk me through what steps you're running and how?

There were a couple of oddities with the manual release process that should be resolved now - or soon.

The bracket-match issue: #1750
Anti-tamper was running after publish: #1752
